buildbot-buildmaster/buildbot/secrets/README.md

```diff
@@ -0,0 +1,7 @@
+This is the place for
+- htpasswd file
```

Comment:
Please add a few more details to provide sufficient context

Reply:
The file contains now a more detailed description, including example.